docs/open-source/index.mdx

```diff
@@ -1,8 +1,18 @@
 ---
 title: Open source
-description: TODO - filler text so that the SEO check doesn't complain!
+description: Qiskit open-source introduction
 ---
 
 # Open source
 
-Blah blah
+Since 2016, IBM Quantum has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
+This groundbreaking initiative includes open-sourcing key parts of the quantum software stack and introducing Qiskit, a versatile quantum computing framework.
+Today, Qiskit is the most popular quantum framework, thanks to its vibrant community.
+The Qiskit community is a dynamic and welcoming group of developers, scientists, and enthusiasts collaborating to advance software, research, education, and business through quantum computing.
+By making quantum computing open and accessible, we accelerate innovation and discover new applications for this transformative technology.
+
+As quantum technology evolves, contributions from the open-source Qiskit Ecosystem projects and developers are crucial in shaping its future.
+The development of a dense ecosystem is vital not only for extending Qiskit's capabilities but also for spreading involvement and engagement across various sectors.
+By joining the Qiskit community, you can participate in the development of quantum technologies.
+Whether you are a seasoned quantum programmer or just beginning your quantum journey, Qiskit provides the support and inspiration needed to push boundaries.
+We welcome all kinds of contributions, from code in PRs to community support on Slack, ensuring everyone can play a role in advancing quantum technology.
```

Comment:
```suggestion
We welcome all kinds of contributions, from code in PRs to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

Reply:
should we say "pull requests" instead of PRs? in case people are unfamiliar with the acronym

Reply:
That was a bit my point, "from expert to not". _PR_ is talking to the expert. But maybe it is trying too hard to look too clever.

Reply:
> PR is talking to the expert.

No need to gatekeep. Even if beginners won't be contributing PRs, it's good for them to be able to understand the document. For example, maybe it helps to learn what PRs are so they in several months start contributing.